src/test/java/org/cactoos/io/CloseShieldOutputStreamTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.io;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import org.cactoos.text.TextOf;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.IsText;
+
+/**
+ * Test case for {@link CloseShieldOutputStream}.
+ * @since 1.0.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+final class CloseShieldOutputStreamTest {
+
+    @Test
+    void writesContentToFile(@TempDir final Path tempdir) throws IOException {
+        final File file = new File(tempdir.toFile(), "cactoos-1.txt-1");
+        file.createNewFile();
+        final Path temp = file.toPath();
+        try (OutputStream out = new OutputStreamTo(temp)) {
+            new Assertion<>(
+                "Must copy Input to Output and return Input",
+                new TextOf(
+                    new Sticky(
+                        new TeeInput(
+                            new ResourceOf("org/cactoos/small-text.txt"),
+                            new OutputTo(new CloseShieldOutputStream(out))
+                        )
+                    )
+                ),
+                new IsText(
+                    new TextOf(temp)
+                )
+            ).affirm();
+        }
+    }
+
+    @Test
+    @SuppressWarnings("try")
+    void preventsOriginalStreamToBeClosed() throws Exception {
+        try (FakeOutputStream origin = new FakeOutputStream()) {
```

Comment:
@baudoliver7 Here it should be `OutputStream` on the left side, and in the class below as well

Reply:
@andreoss Ditto.